android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/BrightnessModule.java

```diff
@@ -71,10 +97,85 @@ public void setSystemBrightnessAsync(final float brightnessValue, final Promise
         );
         promise.resolve(null);
       } else {
-        promise.reject("E_BRIGHTNESS_PERMISSIONS", "WRITE_SETTINGS not granted");
+        promise.reject("ERR_BRIGHTNESS_PERMISSIONS", "WRITE_SETTINGS not granted");
```

Comment:
Wondering if/when we want to have error codes that describe categories (ex: brightness permissions) vs errors (ex: brightness permissions denied). A few thoughts: categories are useful for when we don't know that much about the specific error at hand, like when we wrap the whole function in try-catch. Specific errors are useful for distinguishing between two different errors thrown by the same function (or set of functions you call together in the same JS try-catch block).